packages/orchestration/src/utils/address.js

```diff
@@ -0,0 +1,46 @@
+// @ts-check
+import { Fail } from '@agoric/assert';
+
+/**
+ * @param {import('../types').ConnectionId} hostConnectionId
+ * @param {import('../types').ConnectionId} controllerConnectionId
+ * @param {object} [opts]
+ * @param {'ics27-1' | string} [opts.version] - default is `ics27-1`
```

Comment:
I suspect that the `| string` widens the type such that the literal has no effect. Is that correct?